guides/doc-Configuring_Load_Balancer/topics/Registering_Clients.adoc

```diff
@@ -45,7 +45,7 @@ For more information, see {ManagingHostsDocURL}Registering_a_Host_Using_the_Boot
 <2> Include the `--force` option to register the client that has been previously registered to a standalone {SmartProxy}.
 
 
-* On {RHEL} 7, 6, or 5, enter the following command:
+* On {EL} 7, 6, or 5, enter the following command:
```

Comment:
```suggestion
* On {EL} 5, 6, or 7, enter the following command:
```

Reply:
I'd rather keep this PR focused and not change too many things.